Components/FlightControl/RocketSM.cpp

```diff
@@ -276,7 +276,7 @@ RocketState Fill::OnEnter()
     // Clear the arm flags
     for (uint8_t i = 0; i < 2; i++)
         arrArmConfirmFlags[i] = false;
-
+    GPIO::PowerSelect::UmbilicalPower();
```

Comment:
I'm not too familiar with the current revision of the PMB and how we perform power switching, but this implies that we're always connected to ground power (received over umbilical) before flight.

Does this hold true during testing (ie. do we still have power if we're only using onboard batteries but switching Bat_Enable off)? We haven't ever tested with the umbilical and I'm worried that keeping default-ground power while testing is the wrong move. (gotta ask hardware if the PMB prioritizes rocket power over the Bat_Enable signal)

Reply:
Yep you're right, we should figure out how we should handle this and obviously test with the umbilical when we can. During testing with only batteries and bat_enable off we still do have power and everything worked.

Reply:
Deleting for now while we confirm with hardware / rest of the team.